JSDemos/Demos/DateBox/Formatting/description.md

```diff
@@ -1 +1,17 @@
-This demo shows how to&nbsp;use a&nbsp;locale-dependent format (or&nbsp;default format), <a href="/Documentation/ApiReference/Common/Object_Structures/format/#type">built-in format</a>, <a href="/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String">LDML&nbsp;pattern</a>, and a&nbsp;format with literal characters to&nbsp;specify the <a href="/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#displayFormat">displayFormat</a>. Note that when the <a href="/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#useMaskBehavior">useMaskBehavior</a> property is&nbsp;set to&nbsp;<b>true</b>, the <b>DateBox</b> formats the display value and also controls user input.
\ No newline at end of file
+To customize how a **DateBox** displays dates, use the [displayFormat](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#displayFormat) property. This demo illustrates the following use cases:
+
+- **Locale-dependent format**     
+The DateBox formats its value according to the user's locale. This is the default behavior. You don't need to define the **displayFormat** property. Refer to the following help topic for more information: [Localize Dates, Numbers, and Currencies](/Documentation/Guide/Common/Localization/#Localize_Dates_Numbers_and_Currencies)
+
+- **Built-in predefined formats**    
+DevExtreme supports a number of [predefined formats](/Documentation/ApiReference/Common/Object_Structures/format/#type). Those are simple strings or shortcuts that define commonly used date formats so that you don't need to spell out a complex expression. Review the linked document for the list of values you can assign to **displayFormat**.  This demo uses the "shortdate" format. 
+
+- **LDML pattern**    
+Use an LDML pattern to construct a custom date-time format string. This demo sets **displayFormat** property to "EEEE, d of MMM, yyyy HH:mm" - a combination of day of the week, day number, month, year, hour, and minute. You can see the formatted value in the editor. Refer to the following help topic for information about supported format characters: [Custom Format String](/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String)    
+
+- **Format with literal characters**    
+Specify the **displayFormat** property as a string that contains literal characters. This string can also contain LDML format characters. Wrap any characters that are not a part of the LDML pattern in quotation marks. Otherwise they may be interpreted as wildcards. In this example, if you remove quotation marks from the word "Year", the letter "a" will be interpreted as "AM/PM" placeholder.
```

Comment:
```suggestion
Specify the **displayFormat** property as a string that contains literal and LDML characters. Wrap any characters that are not part of the LDML pattern in quotation marks. Otherwise, they can be interpreted as wildcards. In this example, if you remove quotation marks from the word "Year", the letter "a" is interpreted as an "AM/PM" placeholder.
```